pages/agent/v3/help/_env_unset.md

```diff
@@ -0,0 +1,53 @@
+<!--
+  _____   ____    _   _  ____ _______   ______ _____ _____ _______
+ |  __ \ / __ \  | \ | |/ __ \__   __| |  ____|  __ \_   _|__   __|
+ | |  | | |  | | |  \| | |  | | | |    | |__  | |  | || |    | |
+ | |  | | |  | | | . ` | |  | | | |    |  __| | |  | || |    | |
+ | |__| | |__| | | |\  | |__| | | |    | |____| |__| || |_   | |
+ |_____/ \____/  |_| \_|\____/  |_|    |______|_____/_____|  |_|
+
+This file is auto-generated by scripts/update-agent-help.sh, please update the
+agent CLI help in https://github.com/buildkite/agent and run the generation
+script.
+
+-->
+
+### Usage
+
+`buildkite-agent env unset [variables]`
+
+### Description
+Un-sets environment variables in the current job execution environment. 
```

Comment:
What do you think about removing the hyphen here?

```suggestion
Unsets environment variables in the current job execution environment. 
```

Reply:
lol: 
<img width="590" alt="CleanShot 2023-03-21 at 14 24 28@2x" src="https://user-images.githubusercontent.com/15753101/226498664-17a0c8cd-7460-4bca-8a59-a4adbe372e0a.png">